easybuild/easyconfigs/o/ONNX/ONNX-1.15.0-gfbf-2023a.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'PythonPackage'
+
+name = 'ONNX'
+version = '1.15.0'
+
+homepage = 'https://onnx.ai'
+description = """
+Open Neural Network Exchange (ONNX) is an open ecosystem that empowers AI
+developers to choose the right tools as their project evolves. ONNX provides an
+open source format for AI models, both deep learning and traditional ML. It
+defines an extensible computation graph model, as well as definitions of
+built-in operators and standard data types. Currently we focus on the
+capabilities needed for inferencing (scoring)."""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['ONNX-1.15.0_fix-protobuf-linking.patch']
+checksums = [
+    {'onnx-1.15.0.tar.gz': 'b18461a7d38f286618ca2a6e78062a2a9c634ce498e631e708a8041b00094825'},
+    {'ONNX-1.15.0_fix-protobuf-linking.patch': '5c11eca5275a25d69f989571fe1f387b84c7334478356577f6f6349c5954f54d'},
+]
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('setuptools', '69.1.0'),
```

Comment:
there is a comment on the easyconfig for 2022b saying 

`# more recent setuptools required to fix: AttributeError: editable_mode`, 

and then version 64.0.3 was used (the `setuptools` from the `Python` easyconfig for 2022b was version 63.4.3)

But in 2023a the version of `setuptools` is 67.7.2, so is it still necessary to use an even more recent one here?